h2o-py/h2o/frame.py

```diff
@@ -3831,17 +3831,19 @@ def table(self, data2=None, dense=True):
             expr=ExprNode("table", self, dense))
 
 
-    def hist(self, breaks="sturges", plot=True, **kwargs):
+    def hist(self, breaks="sturges", plot=True, save_plot_path=None, **kwargs):
```

Comment:
interesting, strange use of `kwargs` in this method, it is used for a param that is usually public (`server`), I guess we'll never know why :)